### DISCUSSION TITLE
can not use on miui 14

### DISCUSSION THREAD

**Discussion by yefeixiang** [LSPosed/LSPosed#2315] [Q&A] [answered]
can not use on miui 14,riru and zygisk.

when I use lsposed 1.8.5,I can not  power on 

miui system version  14.0.6.0.TLKCNXM
K50 pro

thanks.

**Comment by yefeixiang** [comment, marked as answer]
面具旧模块跟miui14冲突了，可以试试把所有模块除了救砖都删了然后重刷lsposed

**Comment by Johnny520** [comment]
The old mask module conflicts with miui14. You can try to delete all the modules except the rescue bricks and then repaint lsposed
